ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstringTests/LongestSubstringTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using LeetCodeTraining.LongestSubstring;
+
+namespace LeetCodeTraining.Tests.LongestSubstringTests
+{
+    public class LongestSubstringTests
+    {
+        [Fact]
+        public void LengthOfLongestSubstringSingleChar()
+        {
+            //Arrange
+            string input = "a";
+            int expectedResult = 1;
+
+            //Act
+            int result = LongestSubstringSolution.LengthOfLongestSubstring(input);
+
+            //Assert
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void LengthOfLongestSubstringDoubleChar()
+        {
+            //Arrange
+            string input = "ab";
+            int expectedResult = 2;
+
+            //Act
+            int result = LongestSubstringSolution.LengthOfLongestSubstring(input);
+
+            //Assert
+            Assert.Equal(expectedResult, result);
+        }
+
+        [Fact]
+        public void LengthOfLongestSubstringMultipleCharsOne()
```

Comment:
Please try to use [Theory] and [InlineData] attributes for the xUnit test and rewrite it to get rid of duplicated code in the tests.
Hint: in case of using suggested attributes you will have 4 attributes InlineData before the test and don't have Arrange block in the test, actually arrange block will be moved to in test's parameters (this is for case of implementing 3 leetcode testcases in your case it can be more).
https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/